src/backends/torch/torchlib.cc

```diff
@@ -1044,6 +1110,9 @@ namespace dd
           }
         else if (_timeserie)
           {
+            if (_module._native != nullptr)
+              output = _module._native->cleanup_output(output);
+
```

Comment:
Is the call to `cleanup_output` specific to timeseries or can we put it outside of the "if"?

Reply:
done